.github/workflows/autoupdate.yaml

```
@@ -0,0 +1,60 @@
+# Configuration: https://dependabot.com/docs/config-file/
+# Docs: https://docs.github.com/en/github/administering-a-repository/keeping-your-dependencies-updated-automatically
+
+
+version: 2
```

actionlint (1.7.4)

unexpected key "version" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)